Makefile

```diff
@@ -198,6 +205,11 @@ test-prove: $(proof_tests:=.prove)
 
 test-klab-prove: $(quick_proof_tests:=.klab-prove)
 
+### Official test suite
+
+official_tests:=$(wildcard tests/official/supported/*.wast)
+test-official: $(official_tests:=.run-term)
```

Comment:
This is conceptually duplicate of the `Conformance` block above.

Instead, rename `parse-conformance` to `test-parse-conformance`, and then add targets `test-conformance-supported` and `test-conformance-unsupported` (so it's easy to run either). Also add `test-conformance` which runs both `test-conformance-supported` and `test-conformance-unsupported`.

Reply:
Yes, but it's slow. I thought we were aiming to have a list of unsupported files, so better to do like in KEVM and maintain those files. Then the script makes sense since it updates those files. It could be made a make target, but I'd prefer if the `make` target doesn't modify committed files.